### ISSUE TITLE
allow pipe on newline with backslash

### ISSUE THREAD

**Issue by coderofsalvation** [coderofsalvation/powscript#22]

**Comment by fcard** [comment]
You mean like:
```sh
f x\
| g y
``` 
?
It's really useful, but I wonder what whitespace rules should be applied to it, if any... 
I was thinking about that, since we're removing symbols and making things name-based, it maybe would be nice to have, at least in addition to the above, something like:
```
pipe
  f x
  g y
```
As well as other things like:
```
subshell
  f x
  g y
```
```
fork
  f x
  g y
```

Regardless, escaped newlines are a neat feature to have, in this and other contexts.

**Comment by fcard** [comment]
Implemented in #38 
![slashc](https://user-images.githubusercontent.com/12222927/40871146-7c16cfdc-660c-11e8-8bd8-ca8d9b8f4d6f.gif)
